docs/docs/concepts/auth.md

```diff
@@ -143,6 +143,54 @@ The returned user information is available:
     In many of our tutorials, we will just show the "authorization" parameter to be concise, but you can opt to accept more information as needed
     to implement your custom authentication scheme.
 
+### Agent authentication
+
+When you run an agent on LangGraph Platform (cloud or self-hosted), the platform handles authentication for every request through authentication middleware.
```

Comment:
Technically not correct for standalone-container deployments (see the preceding content in this doc). Your purpose in this section is NOT to re-hash what's already been said, it's to introduce the concept of agents running on-behalf-of users. This is only really relevant for users who have set up custom auth.

Maybe something lke

```suggestion
Custom auth permits delegated access. The values you return in  `@auth.authenticate` are added to the run context, giving agents user-scoped credentials lets them access resources on the user’s behalf.
````